src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,39 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/cars/add")
+public class AddDriverToCarController extends HttpServlet {
```

Comment:
URL says: you add car. Class name says: you add driver to car.
/cars/drivers/add maybe